### DISCUSSION TITLE
Access getServerSideProps in the getServerSideProps?

### DISCUSSION THREAD

**Discussion by powerczuk** [vercel/next.js#17517] [Help] [answered]
Hi guys, just a short question.

I need to create global configuration object based on the domain /en/de/it and access it while fetching the data in the getServerSideProps method.

My idea was to create it in the _app wrapper and provide as a props to the other components.

The question is if it's possible to access the props in the getServerSideProps? Do you have any other idea how I could solve that?

**Comment by karanpvyas** [comment, marked as answer]
Create a utility that would generate your configuration object and ->
i) use it within _app wrapper if your components need it
ii) use it within getServerSideProps (from context.req object, you can find the domain) to create that object.

Hope this helps

**Reply by powerczuk** [reply]
This is what I'm doing at the moment, but based on the domain I'm calculating other params and lot's of code is duplicated on the each page.
